Extension/package.json

```diff
@@ -6311,7 +6311,7 @@
         "@types/minimatch": "^3.0.5",
         "@types/mkdirp": "^0.5.2",
         "@types/mocha": "^10.0.1",
-        "@types/node": "^18.11.9",
+        "@types/node": "^18.15.0",
```

Comment:
VSCode has updated to node 18.15.0 bindings

Reply:
Does this work with the minimum version of VS Code that we support? I don't remember if these need to be in sync or not.

    "engines": {
        "vscode": "^1.67.0"
    },

Reply:
Yes, in that `18.11.9` was technically newer than the node specified in vscode `1.67` too, but we don't use APIs that aren't available.  (and older node bindings break with the newer TS builds)

The difference between `18.11.9` and `18.15.0` typings is very minor (and when we compile, TS typings are gone at runtime regardless) 

So, we're no less compatible than before, but we do get to compile with greater conformance to the API now.